src/hooks/useAlert.js

```diff
@@ -0,0 +1,13 @@
+import { useAlert as useRuntimeAlert } from '@dhis2/app-service-alerts'
+import { useMemo } from 'react'
+import { ALERT_CRITICAL } from '../constants/alerts.js'
+
+function useAlert({ message, severity = ALERT_CRITICAL }) {
+    const openMapErrorAlert = useRuntimeAlert(message, severity)
+
+    const alert = useMemo(() => openMapErrorAlert, []) // eslint-disable-line react-hooks/exhaustive-deps
+
+    return alert
+}
```

Comment:
I agree a hack like this should not be needed, and an investigation of why this is needed in this case is needed. It could be that this app is using the hook differently than anticipated, and we need to assess whether changes need to be made in the app and/or in the hook (and its docs).

There also seems to be something wrong with this custom hook: It is providing the wrong function signature to `useRuntimeAlert`, which accepts a string as first argument and an options object as second argument ([see docs](https://developers.dhis2.org/docs/app-runtime/hooks/useAlert)), but here the second argument is a string.

It's also not really clear to me how the fix above relates to what you wrote in the PR description:
> When using the app-runtime useAlert directly in useLoadMap, the map was getting re-rendered when opening the interpretations panel and when going to download mode.

All the custom hook above does is memoize the `show` function returned from the hook. And this `show` function is wrapped in a `useCallback`, which should make it a fairly stable reference. Perhaps you could explain this a bit more.

Reply:
I've removed the useAlert wrapper. This can be investigated later. I've removed all the alerts, as this was a bit of scope creep anyway.